src/api/app/models/event_find_subscriptions.rb

```diff
@@ -8,18 +8,18 @@ def subscriptions
     @payload = @event.payload
     @subscriptions = EventSubscription.where(eventtype: @event.class.classnames)
 
-    # 1. generic defaults
+    # Get the defaults
+    # FIXME: These defaults can only be created by manually adding EventSubscription
+    #        to the database. We don't do that anywhere in the code base. Either
+    #        we'll do an interface for it or set the default somewhere sane...
```

Comment:
I see you are following the same path I went down the event_subscriptions rabbit hole a few months ago ;)

EventSubscriptions with no user_id or group_id are the global defaults that all new users get created with until they update their own settings. They are set by the admin at [`Webui::SubscriptionsController`](https://github.com/evanrolfe/open-build-service/blob/master/src/api/app/controllers/webui/subscriptions_controller.rb#L8)

Please update this comment.

Reply:
> rabbit hole

With the difference that I try to draw a map so the next guy finds in/out easier ;-)

Reply:
> With the difference that I try to draw a map so the next guy finds in/out easier ;-)

Not so different! I refactored a lot of the EventSubscription code, if you think it is bad now you should have seen it before!

Reply:
hahahahaha 🤣